src/test/tests/queries/queriesOverTime.py

```diff
@@ -339,7 +341,7 @@ def TestTransientVariable():
 
     pick = GetPickAttributes()
     pick.doTimeCurve = 1
-    pick.timePreserveCoord = 0
+    pick.timePreserveCoord = 1
```

Comment:
Though setting this may preserve the old QOT route, it does not perform the same query. The results are different.

Reply:
Since the results drastically change, and since pick currently doesn't utilize the 'actual data' flag except with your changes to trigger the new route, I prefer you use that flag to preserve old route instead of changing the behavior of the test.

Reply:
I just committed a change that reflects this. I updated the ViewerQueryManager to check for the "use_actual_data" flag early on, and I changed the PickByNode calls to not use the deprecated versions. I did notice that the use_actual_data flag isn't being passed on to the time query when using PickByNode, though. Is there a reason for this?

Reply:
Pick has always been an 'actual data' only query up until now, so the flag hasn't been needed.  Since all current tests assume 'actual data', then any use of the flag in old tests should set it to 'true'.

Reply:
Do we want the default of PickThroughTime to still use actual data? If so, I will probably need to add an actual vs original flag in the GUI.

Reply:
Okay, I've added the following which should resolve these issues:

- Added option to set the use_actual_data flag when performing a query through time from the Query menu GUI (excluded from the Pick menu GUI).
-Add checks to capture the use_actual_data flag from python calls.
-If the flag is not set, it will be assumed to be true.